### PR TITLE
add new geth flags

### PR DESCRIPTION
**Pull request by cdetrio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,7 +162,7 @@ Aleth supports EVMC out of the box, but geth not yet.
 	Run the built geth with configuration for ewasm testnet.
 
 	```sh
-	./build/bin/geth --datadir /tmp/ewasm-node/4201/ --etherbase 031159dF845ADe415202e6DA299223cb640B9DB0 --rpc --rpcapi "web3,net,eth,debug" --rpcvhosts="*" --rpcaddr "0.0.0.0" --rpccorsdomain "*" --vmodule "miner=12,rpc=12" --mine --miner.threads 1 --nodiscover --networkid 66 
+	./build/bin/geth --datadir /tmp/ewasm-node/4201/ --etherbase 031159dF845ADe415202e6DA299223cb640B9DB0 --vm.ewasm="/home/ewasm/hera/src/libhera.so:metering=true:fallback=true" --rpc --rpcapi "web3,net,eth,debug" --rpcvhosts="*" --rpcaddr "0.0.0.0" --rpccorsdomain "*" --vmodule "miner=12,rpc=12" --mine --miner.threads 1 --nodiscover --networkid 66
```
Comment (jwasinger) [inline README.md]: nitpick:  I prefer `/path/to/libhera.so'

**Comment by lrettig** [comment]
Is this still current? @cdetrio @jwasinger @chfast

File: README.md [hunk 2]
```diff
@@ -139,6 +139,7 @@ Aleth supports EVMC out of the box, but geth not yet.
 	```sh
 	> EVMC_PATH=~/hera/build/src/libhera.so EVMC_OPTIONS='metering=true fallback=true' \
 	./build/bin/geth \
+	--vm.ewasm="/home/ewasm/hera/src/libhera.so:metering=true:fallback=true" \
```
[the remark this answers is not in the document]
Comment (lrettig) [reply README.md]: Fixed

File: README.md [hunk 3]
```diff
@@ -137,8 +137,8 @@ Aleth supports EVMC out of the box, but geth not yet.
 	Here's the recommended configuration for connecting your node to the Ewasm testnet:
 
 	```sh
-	> EVMC_PATH=~/hera/build/src/libhera.so EVMC_OPTIONS='metering=true fallback=true' \
-	./build/bin/geth \
+	> ./build/bin/geth \
+	--vm.ewasm="/home/ewasm/hera/src/libhera.so:metering=true:fallback=true" \
```
Comment (axic) [inline README.md]: Actually the `libhera.so` path is wrong. It should be `hera/src/libhera`, but the build dir.

I think the original `~/hera/build/src/libhera.so` was slightly better.

File: README.md (hunk 1 above)
Comment (axic) [reply README.md]: Or this, this may be a better suggestion for both places.

File: README.md (hunk 3 above)
Comment (axic) [reply README.md]: Even better: https://github.com/ewasm/testnet/pull/94#discussion_r228764637

Comment (chfast) [reply README.md]: Use , not :

Comment (lrettig) [reply README.md]: fixed

**Comment by lrettig** [comment]
This is ready to be merged

File: README.md [hunk 4]
```diff
@@ -161,7 +161,7 @@ Aleth supports EVMC out of the box, but geth not yet.
 	Run the built geth with configuration for ewasm testnet.
 
 	```sh
-	./build/bin/geth --datadir /tmp/ewasm-node/4201/ --etherbase 031159dF845ADe415202e6DA299223cb640B9DB0 --rpc --rpcapi "web3,net,eth,debug" --rpcvhosts="*" --rpcaddr "0.0.0.0" --rpccorsdomain "*" --vmodule "miner=12,rpc=12" --mine --miner.threads 1 --nodiscover --networkid 66 
+	./build/bin/geth --datadir /tmp/ewasm-node/4201/ --etherbase 031159dF845ADe415202e6DA299223cb640B9DB0 --vm.ewasm="/path/to/libhera.so:metering=true:fallback=true" --rpc --rpcapi "web3,net,eth,debug" --rpcvhosts="*" --rpcaddr "0.0.0.0" --rpccorsdomain "*" --vmodule "miner=12,rpc=12" --mine --miner.threads 1 --nodiscover --networkid 66
```
Comment (axic) [inline README.md]: This one needs commas too

**Pull request merged**